examples/dataset/age_estimation/age_estimation/upload_dataset.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from argparse import ArgumentParser
+from argparse import Namespace
+
+import pandas as pd
+from age_estimation.constants import DATA_FILEPATH
+from age_estimation.constants import DATASET
+
+import kolena
+from kolena.dataset import upload_dataset
+
+
+def run(args: Namespace) -> None:
+    df = pd.read_csv(DATA_FILEPATH)
+
+    kolena.initialize(verbose=True)
+    upload_dataset(args.dataset, df)
+
+
+def main() -> None:
+    ap = ArgumentParser()
+    ap.add_argument(
+        "--dataset",
+        type=str,
+        default=DATASET,
+        help=f"Custom name for the {DATASET} dataset to upload.",
```

Comment:
Nit: `Optionally specify a custom name`

Reply:
Updated in https://github.com/kolenaIO/kolena/pull/427/commits/d11c0195834987f8ff9e224cad7f0da2fadd0464